pkg/database/duration.go

```diff
@@ -40,6 +40,10 @@ func FromDuration(d time.Duration) DurationSeconds {
 	}
 }
 
+func (d *DurationSeconds) Days() int64 {
+	return int64(d.Duration.Hours() / 24)
```

Comment:
Hours returns a float, so it's probably fine, but I'd feel better if we divided by 24.0 to avoid potential integer division if Go changes this API 10 years from now.